bread/querysetfield/__init__.py

```diff
@@ -58,7 +58,7 @@ def deconstruct(self):
         name, path, args, kwargs = super().deconstruct()
         return name, path, args, {"modelfieldname": self.modelfieldname, **kwargs}
 
-    def contribute_to_class(self, cls, name):
+    def contribute_to_class(self, cls, name, **kwargs):
         super().contribute_to_class(cls, name)
```

Comment:
If we take kwargs we should also pass them to the call to super. Otherwise they would get silently lost.

Reply:
I see. I changed this one so that it passes `**kwargs` to `super().contribute_to_class(cls, name, **kwargs`.